datadog_cluster_agent/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## 5.5.0 / 2025-02-20
```

Comment:
does the agent version get added after it is released?

Reply:
@jennchenn yup! see this for example:

https://github.com/DataDog/integrations-core/pull/19640